site/js/editor.js

```diff
@@ -209,11 +209,13 @@ NFAEditor.prototype = {
         }
         return true;
     },
-    gotoStep: function( input, step ) {
+    gotoStep: function( input, steps ) {
```

Comment:
Each call of the `gotoStep` makes a run of the automaton from the initial state. I read this methods as follows: given the `input` it goes to a step following the provided number of `steps`. The user (invoker) recieves an actual step (which may deffer from the desired one) where he (she) found himself after the invocation. 
However, I do agree that all the code may be moved to the `run` method of the editor. In this way `run(input, steps)` or `run(input, length)` would sound more clear.